app/src/main/java/ca/dal/cs/athletemonitor/athletemonitor/AccountManager.java

```diff
@@ -16,6 +16,12 @@
  * This class performs management of a user account
  */
 public class AccountManager {
+
+    /**
+     * lastAuth holds the username of the last authenticated user
+     */
+    private static String lastAuth = "";
```

Comment:
I'm not sure if I like this approach, but, given the upcoming iteration deadline and lack of a better quick alternative, it's fine for now.

Reply:
It's almost as secure as plaintext passwords!